test/unit_test/api/apps/sdk/test_sdk_doc_llm_token_mapping.py

```
@@ -0,0 +1,165 @@
+
+
+
+def _load_chunk_api():
+    """Execute ``chunk_api.py`` with the ``manager`` blueprint global that the
+    app loader normally injects, so ``_map_doc`` is importable without booting
+    the API server."""
+    src_path = _APPS_DIR / "restful_apis" / "chunk_api.py"
+    module = ModuleType("_test_chunk_api")
+    module.__file__ = str(src_path)
+    module.manager = SimpleNamespace(route=lambda *a, **k: (lambda fn: fn))
+    exec(compile(src_path.read_text(), str(src_path), "exec"), module.__dict__)
```

Ruff (0.16.3)

[error] Use of `exec` detected

(S102)